exercises/G-conditionals-2/exercise-1.js

```diff
@@ -6,7 +6,12 @@
   - if number is more or equal to zero, return the word "positive"
 */
 
-function negativeOrPositive(number) {}
+function negativeOrPositive(number) {
+    if (number >= 0) {
+        return "positive";
+    }
+    return "negative";
+}
```

Comment:
Good work - you do need the if clause here as you are not returning just true/false.